uniswap/uniswap.py

```
@@ -1224,16 +1275,21 @@ def estimate_price_impact(
 
         See ``examples/price_impact.py`` for an example which uses this.
         """
-        amount_small = 10 ** 2
-        cost_small = self.get_price_input(
-            token_in, token_out, amount_small, fee=fee, route=route
-        )
-        cost_amount = self.get_price_input(
-            token_in, token_out, amount_in, fee=fee, route=route
-        )
-
-        price_small = cost_small / amount_small
-        price_amount = cost_amount / amount_in
+        try:
+            price_small = self.get_raw_price(
+                token_in, token_out, fee=fee,
+            )
+        except:
+            return 1
+        if price_small == 0:
+            return 1
+        try:
+            cost_amount = self.get_price_input(
+                token_in, token_out, amount_in, fee=fee, route=route
+            )
+        except:
```

Bare except here again